frontend/tests/App.spec.js

```diff
@@ -82,20 +82,56 @@ describe('App.vue', () => {
     };
     const collection1 = {
         id: "id1",
-        title: "title1",
+        title: "colTitle1",
         description: "description1",
         itemIds: [],
         type: 0,
         items: [item2]
     };
+    const notExistsItemFqid = "extName1.extPublisher1.null";
     wrapper = initComponent(App);
     wrapper.vm.showCollections([collection1]);
-    wrapper.vm.changeItemsState([{fqid:fqid1, readState: "READ"}]);
-    const tarItem = wrapper.vm.getItemByFqid(fqid1);
-    expect(tarItem.readState).toEqual("READ");
+    const ret = wrapper.vm.changeItemsState([{fqid:notExistsItemFqid, readState: "READ"}]);
+    expect(ret).toBeFalsy();
```

Comment:
`changeItemsState` is always falsly...

Reply:
we want to add assertions of this style
```javascript
wrapper.vm.showCollections([collection1]);
// SAVE STATE
const beforeState = structuredClone(warpper.vm.collections)
wrapper.vm.changeItemsState([{fqid:notExistsItemFqid, readState: "READ"}]);
// ASSERT STATE has not changed
const afterState = structuredClone(warpper.vm.collections)
expect(beforeState).to.deep.equal(afterState)
```
- This could be useful to save the state: structuredClone() (look in MDN)


Option 2:
We modify `changeItemsState` to return a boolean if if actually performed a change.


Y